content/docs/0.11.x/operate/advanced_install_options/index.md

```diff
@@ -27,35 +27,7 @@ helm install helm-service https://github.com/keptn/keptn/releases/download/0.11.
-| `control-plane.mongodb.adminPassword`                 | If an external MongoDB instance is used, specify the admin password to access the MongoDB                                                                                       | any string                                                                                       | empty                                 |
-| `control-plane.ingress.enabled`                       | Allows you to Install your own ingress object for Keptn                                                                                                                         | true,false                                                                                       | false                                 |
-| `control-plane.ingress.annotations`                   | If ingress is set to true you can optionally specify any annotations you require                                                                                                | any string                                                                                       | empty                                 |
-| `control-plane.ingress.host`                          | If ingress is set to true you can optionally specify a host                                                                                                                     | any string                                                                                       | empty                                 |
-| `control-plane.ingress.path`                          | If ingress is set to true you can specify a path                                                                                                                                | any string                                                                                       | /                                     |
-| `control-plane.ingress.pathType`                      | If ingress is set to true you can specify a pathType                                                                                                                            | any string                                                                                       | Prefix                                |
-| `control-plane.ingress.tls`                           | If ingress is set to true you can optionally specify tls config                                                                                                                 | any string                                                                                       | empty                                 |
+The list of available flags are available [here](https://github.com/keptn/keptn/tree/master/installer/manifests/keptn).
```

Comment:
Please *never* refer to a document in the `master` branch, always use the release tag or release branch.